src/index.css

```diff
@@ -0,0 +1,140 @@
+@import url("./styles/blocks/card.css");
+@import url("./styles/blocks/stars.css");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  color: #060b35;
+}
+
+* {
+  margin: 0;
+}
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  padding: 31px 15px 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
```

Comment:
why do you describe `.card` when you already imported it? 🤔

Reply:
Yes i have problem with deploy of this task.
I imported it in third commit only because i'm trying to fix npm deploy malfunction.